.github/workflows/on-push-master-publish-chart.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish helm chart
+
+on:
+  push:
+    branches:
+      - master
+      - main # assume that the branch name may change in future
+    paths:
+      - 'charts/**' 
+      - '.github/**'     
+env:
+  KUBE_SCORE_VERSION: 1.10.1
```

Comment:
```suggestion
```

Reply:
I'm not sure I understand the suggestion here, the suggest change seems to match the existing content?

Reply:
We can remove the env variable since we never use it in this action.